src/view/invoice.js

```diff
@@ -7,16 +7,24 @@ import MainContent from '../component/main-content';
 import { InputField, AmountInputField } from '../component/field';
 import { Header, Title } from '../component/header';
 import { CancelButton, PillButton, Button } from '../component/button';
-import { BalanceLabel, BalanceLabelUnit } from '../component/label';
+import {
+  BalanceLabel,
+  BalanceLabelNumeral,
+  BalanceLabelUnit,
+} from '../component/label';
 import Card from '../component/card';
 import Icon from '../component/icon';
 import { FormStretcher, FormSubText } from '../component/form';
-import { color } from '../component/style';
+import { fiatToSatoshis, toAmount } from '../helper';
+import { color, font } from '../component/style';
 
 const styles = StyleSheet.create({
   balance: {
     marginTop: 40,
   },
+  fiatUnit: {
+    color: color.blackDark,
+  },
```

Comment:
Also not needed for now.

Reply:
For me, when I take this styling out it looks like this:
![deepinscreenshot_select-area_20180723135918](https://user-images.githubusercontent.com/10647624/43102693-9f5a3a8c-8e80-11e8-9a94-7d75bd3dec23.png)